icon/iiss/icstate/timer.go

```diff
@@ -114,12 +111,12 @@ func (t Timer) IsEmpty() bool {
 
 func (t *Timer) Set(other *Timer) {
 	t.checkWritable()
-	t.Height = other.Height
 	t.Addresses = other.Addresses.Clone()
 }
 
 func (t *Timer) Add(address module.Address) {
 	t.Addresses = append(t.Addresses, address.(*common.Address))
```

Comment:
It would be better to use "common.AddresstoPtr"
If you want to manage it in set, then it should check whether it's already included or not before addition.
Then you can stop at the first match on removal